packages/platform/platform-http/src/common/services/PlatformAdapter.ts

```
@@ -1,23 +1,22 @@
 import {Type} from "@tsed/core";
 import {injectable, ProviderOpts} from "@tsed/di";
-import {PlatformContextHandler, PlatformHandlerMetadata, PlatformLayer} from "@tsed/platform-router";
+import {PlatformHandlerMetadata, PlatformLayer} from "@tsed/platform-router";
 import {IncomingMessage, ServerResponse} from "http";
```

Biome (1.9.4)

[error] A Node.js builtin module should be imported with the node: protocol.

Using the node: protocol is more explicit and signals that the imported module belongs to Node.js.
Unsafe fix: Add the node: protocol.


(lint/style/useNodejsImportProtocol)